src/wallet/wallet.cpp

```diff
@@ -3489,6 +3492,9 @@ bool CWallet::CreateTransaction(
                     }
                 }
 
+                // add csw input 
```

Comment:
See the code above related to the `mempooltxinputlimit` check. We should consider CSW inputs as well.